.circleci/config.yml

```diff
@@ -719,7 +748,7 @@ workflows:
           pg_major: 15
           image_tag: '<< pipeline.parameters.pg15_version >>'
           requires: [build-flaky-15]
-          make: '<< pipeline.parameters.flaky_test_make >>'
+          schedule: '<< pipeline.parameters.flaky_test_schedule >>'
```

Comment:
I don't think we need this anymore, because of the auto-detection done by `run_test.py`. The original reason for `flaky_test_make` was because we didn't have such auto-detection.